src/providers/export/exportProviderFactory.ts

```diff
@@ -13,14 +20,15 @@ export class ExportProviderFactory {
 
```

Comment:
I'd recommend we rename `handlers` to `providers` to be consistent with the other factories that follow a similar model.